pages/blog/asyncapi-ambassador-program.md

```diff
@@ -0,0 +1,66 @@
+
+- Speak at community events or write technical content (e.g., blog posts) that solves community needs.
+
+- Foster community growth and a welcoming environment to spread knowledge about the project.
+
+## Why create an ambassador program?
+
+As we know, each person's way of looking at things is unique. Each person's abilities differ. Following that line, each ambassador can offer a different approach. They bring part of their essence into the content or the program where they are involved and disseminate. This is what brings human value enriching the conception and possibilities of the project.
+
+<p align="center">
+  <img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExOTgzODZhZDM1MWZhMDJiODNkODFhZTI3MWRjZDExZGQyYzBlNzMwYiZjdD1z/NfBApd81ooX87Od5hk/giphy.gif" alt="heart&brain" />
+</p>
+
+We are in a time of fast expansion, which means that everything is moving very fast in the community. Last year we not only started the ambassador program, but also the AsyncAPI Mentorship program and participated in the GSoD for the first time. So, the needs of the community are changing and the initiative react being dynamic and open, driving new ideas and proposals.
+
+## How does the program work?
+
+Like any other management issue related to the initiative, the Ambassador Program is approached as a transparent system, where diversity, inclusiveness, and integration are key concepts that are wanted to be materialized in the program.
+
+<p align="center">
+  <img src="https://media.giphy.com/media/eEh1lwAYtnmwDWSirZ/giphy.gif" alt="heart&brain" />
+</p>
+
+Thus, the process of recruiting ambassadors is completely open. If a person meets the requirements, they only have to add themselves to this [document](https://github.com/asyncapi/community/blob/master/AMBASSADORS_MEMBERS.json), attaching their personal information along with their contributions. The TSC members will then approve the proposal. 
+
+Subsequently, all ambassador information is displayed on the AsyncAPI website once the PR has been merged. Like this, the process try to be as simple, collaborative, and transparent as possible.
+
+## More about AsyncAPI Ambassador Program
+
+The minimum period of enrollment in the program is one year, with the possibility of renewal on an annual basis. At the end of each period, the number of participants will be determined and verified on a yearly basis. The minimum requirement to be an ambassador is to make four contributions per year in the form of articles, talks, videos, podcasts, presentations, drive initiatives, etc.
+
+Ambassadors acquire some smooth amount of responsibilities on holding this degree. In return, to thank the participants for their work, we try to reward their effort with some benefits, such as special swag and free conference tickets.  
+
+If you want to know more about the Ambassador Program, click [here](https://github.com/asyncapi/community/blob/6c066e0eab6c778d1a37d1beebb1418a4bc700f6/AMBASSADOR_ORGANIZATION.md).
```

Comment:
```suggestion
If you want to know more about the Ambassador Program, [see](https://github.com/asyncapi/community/blob/6c066e0eab6c778d1a37d1beebb1418a4bc700f6/AMBASSADOR_ORGANIZATION.md).
```

Reply:
fixed this commit suggestion too because it was:
- too verbose
- the link didn't adhere to accessibility standards; for anchor text links, remember to always use copy that tells a screen reader user where a link takes them (ex: **see** or **here** don't inform me anything at all)